elkm1_lib/elk.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, config, loop=None):
         self._connection_retry_time = 1
         self._message_decode = MessageDecode()
         self._reconnect_task = None
+        self._temperature_units = None
```

Comment:
So, I had just coded this up this week too :). No worries! I'm happy to accept this. Question is, should temperature units be on the Elk class or the Panel class? I put it on the panel class. I also put code length as a property too.

Reply:
> Question is, should temperature units be on the Elk class or the Panel class?

AFAICT the system cannot specify a temp unit per area, only per system.  I did not fully dig though Elk-RP though

Reply:
> So, I had just coded this up this week too :). No worries! I'm happy to accept this

No worries either way how it gets solved.  I'm just be happy to not have to ask the user :)

Reply:
Sorry for the delay, a medical emergency popped up.

Panel is for the entire system, even though the class comment wrongly says area. Area is for the areas. So temperature units should be on panel.

Reply:
Since I have it coded and tested as an attribute on panel I'll go ahead and push my change. Thank you though! You're a coding machine!